camelot/admin/admin_route.py

```diff
@@ -23,14 +23,13 @@ class RouteWithRenderHint(DataclassSerializable):
     route: Route
     render_hint: RenderHint
 
-
 class AdminRoute(object):
     """
     Server side register of admins being used on the client side.
     """
 
     _admin_counter = itertools.count()
-    _admin_routes = NamingContext()
+    _admin_routes = initial_naming_context.bind_new_context('admin')
 
     @classmethod
     def admin_for(cls, route):
```

Comment:
weet ook nt of me die admin_for method eigenlijk nog nodig hebben

waar de admin nodig is, kan die in principe nu via de initial_naming_context w opgevraagd ...